kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -922,7 +927,12 @@ private void convert(Map<String, Boolean> attributes, Att att) {
             Object val = attribute._2;
             String strVal = val.toString();
             sb.append(conn);
-            if (attributes.get(name) != null && attributes.get(name)) {
+            if (cls.equals(K.class)) {
```

Comment:
Do you mean here to say that an attribute of K class will be printed even if it's not in the `attributes` map?  
I would like a little more context on what this PR actually does.